libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -581,9 +580,6 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
 
     if (erase_all_storage) {
         AP_Param::erase_all();
-#if HAL_LOGGING_SITL_ENABLED
-        unlink(AP_Logger_SITL::filename);
```

Comment:
We still need to do this; there's a method on halsitl which allows you to check if you need to clear storage.

Reply:
See https://github.com/ardupilot/ardupilot/blob/master/libraries/SITL/SIM_RAMTRON.cpp#L19

Reply:
Done!

Reply:
As discussed on discord, I have removed this from here and handled this in `init()` method of JEDEC.

Reply:
removed it now